CMakeLists.txt

```diff
@@ -32,12 +36,30 @@ set(HAVE_MMAL ${WITH_MMAL})
 set(HAVE_MYSQL ${WITH_MYSQL})
 set(HAVE_PGSQL ${WITH_PGSQL})
 set(HAVE_SQLITE3 ${WITH_SQLITE3})
-check_include_files("linux/videodev2.h" HAVE_LINUX_VIDEODEV2_H)
-if(${WITH_V4L2})
-	set(WITHOUT_V4L2 HAVE_LINUX_VIDEODEV2_H)
-else(${WITH_V4L2})
-	set(WITHOUT_V4L2 OFF)
-endif(${WITH_V4L2})
+
+set(WITHOUT_V4L2 ON)
+  if(CMAKE_SYSTEM_NAME MATCHES "FreeBSD" AND NOT(WITH_PWCBSD))
```

Comment:
I don't think this 'if' needs to be intended?

The prevailing indentation in this file seems to be tabs, so I think it might be preferable to maintain that style (or switch completely to a different style).